### PR TITLE
Add annotation study results page

### PR DESCRIPTION
**Pull request by msmolens**

### CODE REVIEW

**Comment by msmolens** [comment]
@brianhelba This is updated and rebased. Once the endpoint that serves images is in place, the template will need to be updated:
https://github.com/ImageMarkup/isic-archive/blob/9e3f1c918e411da349f298eca2bc2005a33dff3b/web_external/templates/widgets/studyResultsFeatureImagePage.jade#L3
and possibly the endpoint URL updated:
https://github.com/ImageMarkup/isic-archive/blob/9e3f1c918e411da349f298eca2bc2005a33dff3b/web_external/js/views/body/StudyResultsView.js#L266

File: web_external/js/views/body/StudyResultsView.js [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+        }, this).fetch({
+            studyId: this.study.id,
+            userId: this.userId,
+            imageId: this.imageId
+        });
+    },
+
+    render: function () {
+        this.$el.html(isic.templates.studyResultsPage({
+            title: 'Study Results'
```
Comment (brianhelba) [inline web_external/js/views/body/StudyResultsView.js]: What's this variable do? I can't find its usage within `studyResultsPage.jade`.

File: web_external/templates/body/studyResultsPage.jade [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+      label.control-label(for="isic-study-results-user") User
+      select#isic-study-results-select-user-container.form-control.input-sm(disabled)
+        option(selected disabled)
+          | Select...
+
+    .isic-form-group.isic-study-results-form-group
+       #isic-study-results-global-features-container
+
+     .isic-form-group.isic-study-results-form-group
```
Comment (brianhelba) [inline web_external/templates/body/studyResultsPage.jade]: 5 space indent here. I'll fix.

Comment (msmolens) [inline web_external/templates/body/studyResultsPage.jade]: Thanks. I'm trying a different editor and need to work out its configuration to make some things more automatic.

File: web_external/js/views/body/StudyResultsView.js (hunk 1 above)
Comment (msmolens) [inline web_external/js/views/body/StudyResultsView.js]: I changed this page to always pass titles to templates, similar to the other pages (e.g. Featuresets).

**Comment by msmolens** [comment]
@brianhelba PTAL

**Pull request merged**